lldb/unittests/Host/linux/HostTest.cpp

```diff
@@ -12,6 +12,9 @@
 #include "lldb/Utility/ProcessInfo.h"
 #include "gtest/gtest.h"
 
+#include <cerrno>
+#include <sys/resource.h>
```

Comment:
it is ok to include this header file here because this is a linux specific host test.

Reply:
yeah. The cmake file enforces a linux-only build as well. However as @jimingham pointed out someone could plausibly move this into a non-linux build. Maybe I should place guards here and below? The functionality is in POSIX I think and my understanding is that windows would conform to POSIX to some degree. But to be honest I know very little about what happens on the windows side of the fence.

Reply:
Much of this file would work just fine on FreeBSD as well so that might make sense, although I'm not sure what the best structure would be -- maybe much of this file should move to a new lldb/unittests/Host/posix/HostTest.cpp, leaving anything Linux-specific here?

Reply:
This is fine if this is a host test, so nothing needs to be done. We are expecting linux + posix here which is fine.